examples/application/abi/algobank.py

```diff
@@ -0,0 +1,113 @@
+# This example is provided for informational purposes only and has not been audited for security.
+from pyteal import *
+import json
+
+
+@Subroutine(TealType.none)
+def assert_sender_is_creator() -> Expr:
+    return Assert(Txn.sender() == Global.creator_address())
+
+
+# move any balance that the user has into the "lost" amount when they close out or clear state
+transfer_balance_to_lost = App.globalPut(
+    Bytes("lost"),
+    App.globalGet(Bytes("lost")) + App.localGet(Txn.sender(), Bytes("balance")),
+)
+
+router = Router(
+    name="AlgoBank",
+    bare_calls=BareCallActions(
+        # approve a creation no-op call
+        no_op=OnCompleteAction(action=Approve(), call_config=CallConfig.CREATE),
+        # approve opt-in calls during normal usage, and during creation as a convenience for the creator
+        opt_in=OnCompleteAction(action=Approve(), call_config=CallConfig.ALL),
+        # move any balance that the user has into the "lost" amount when they close out or clear state
+        close_out=OnCompleteAction(
+            action=transfer_balance_to_lost, call_config=CallConfig.CALL
+        ),
+        clear_state=OnCompleteAction(
+            action=transfer_balance_to_lost, call_config=CallConfig.CALL
+        ),
+        # only the creator can update or delete the app
+        update_application=OnCompleteAction(
+            action=assert_sender_is_creator, call_config=CallConfig.CALL
+        ),
```

Comment:
Oops, I actually meant to leave this comment on `abi.rst` not on the actual `.py` example

```suggestion
        # approve a creation no-op call
        no_op=OnCompleteAction(action=Approve(), call_config=CallConfig.CREATE),
        
        # approve opt-in calls during normal usage, and during creation as a convenience for the creator
        opt_in=OnCompleteAction(action=Approve(), call_config=CallConfig.ALL),
        
        # move any balance that the user has into the "lost" amount when they close out or clear state
        close_out=OnCompleteAction(
            action=transfer_balance_to_lost, call_config=CallConfig.CALL
        ),
        clear_state=OnCompleteAction(
            action=transfer_balance_to_lost, call_config=CallConfig.CALL
        ),
        
        # only the creator can update or delete the app
        update_application=OnCompleteAction(
            action=assert_sender_is_creator, call_config=CallConfig.CALL
        ),
```

`black` probably disagrees with me, but I find spacing make it a little easier to read